website/server/models/user/methods.js

```diff
@@ -20,15 +20,16 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
-schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, message) {
+schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, options) {
```

Comment:
Okay, use the format in this file above the  creating tasks function: https://github.com/HabitRPG/habitica/blob/develop/website/server/libs/taskManager.js.